api.js

```diff
@@ -15,18 +21,28 @@ const register = async (firstName, lastName, email, password) => {
       "email": email,
       "password": password
    };
-   return await axios.post(baseurl + '/users', data, config);
+   return await axios.post(baseurl + '/users', data, getConfig(auth=false));
 }
 
 const login = async (email, password) => {
    var data = {
       'email': email,
       'password': password
    };
-   return await axios.post(baseurl + '/users/session', data, config);
+   return await axios.post(baseurl + '/users/session', data, getConfig(auth=false));
+};
+
+const getStockHistory = async (ticker, length) => {
+   var config = getConfig();
+   config['params'] = {
+      ticker,
```

Comment:
Ticker is changing to a url parameter instead of query parameter. I recommend we hold off on merging until that is done.

Reply:
👍 Good call.

Reply:
I'm thinking since this PR also has some changes in the SDK versions and stuff that might be important for other branches, I'll make a JIRA task to change the ticker once Nish is done